packages/sku/src/services/vite/helpers/createConfig.ts

```diff
@@ -107,19 +109,19 @@ export const createViteConfig = ({
       esbuildOptions: {
         plugins: [fixViteVanillaExtractDepScanPlugin()],
       },
+      exclude: skuContext.skuConfig.skipPackageCompatibilityCompilation,
     },
     ssr: {
       external: [
-        ...builtinModules,
         '@vanilla-extract/css/adapter',
         'serialize-javascript',
         'used-styles',
         '@sku-lib/vite',
```

Comment:
Could remove the vanilla extract adapter (can't mark entrypoints as external anyway) and used-styles (unused in sku) from here too if you want.